app/src/main/res/values/dimens.xml

```diff
@@ -24,6 +24,15 @@
   <dimen name="grid_35">280dp</dimen>
   <dimen name="grid_40">320dp</dimen>
 
+  <!-- Kickstarter six-point grid keylines -->
```

Comment:
the other option is to use 8pt grid for inbox until we switch everything to 6pt. do you think that's any better than `_new_` grids? 

i defer to yer judgement!